Makefile

```diff
@@ -18,28 +18,30 @@ all: format build test
 
 test:
 	@echo ">> running tests"
-	@$(GO) test -tags test -short -race $(pkgs)
+	$(GO) test -tags test -short -race $(pkgs)
 
-test-integration: build test
-	@./build/integration.sh
+test-integration:
+	./build/integration.sh
 
 format:
 	@echo ">> formatting code"
-	@$(GO) fmt $(pkgs)
+	$(GO) fmt $(pkgs)
 
 vet:
 	@echo ">> vetting code"
-	@$(GO) vet $(pkgs)
+	$(GO) vet $(pkgs)
 
 build:
 	@echo ">> building binaries"
-	@./build/assets.sh
-	@./build/build.sh
+	./build/assets.sh
```

Comment:
The name of the scripts are not really that descriptive. instead we can have these scripts log what they are doing..